data/tools.yaml

```diff
@@ -342,3 +342,6 @@
   url: https://github.com/KittyHeaven/blue_bird
   tags:
     - testing
+- name: esplanade
+  summary: Validate requests and responses against API Blueprint specifications.
+  url: https://github.com/funbox/esplanade
```

Comment:
This is missing a tags section could you please add `tags: []` on the next line. I think the `tools.html` page won't build correctly otherwise.

Reply:
Thanks